labgrid/driver/shelldriver.py

```diff
@@ -61,13 +61,14 @@ def on_activate(self):
             self._inject_run()
         if self.keyfile:
             self._put_ssh_key(self.keyfile)
-        self._run("dmesg -n 1")  # Turn off Kernel Messages to the console
+        # Turn off Kernel Messages to the console
+        self._run("dmesg -n 1")  # pylint: disable=missing-kwoa
```

Comment:
Please disable missing-kwoa and ￼unused-argument for the whole file. The annotations are too noisy.